### DISCUSSION TITLE
Request: Relative Tab Numbers

### DISCUSSION THREAD

**Discussion by konata-san** [kg8m/chrome-show-tab-numbers#38] [Ideas]
Hello. I would like to request an option to switch between normal tab numbers and relative tab numbers. With relative tab numbers the tab numbers would be rendered as the distance in tabs from the current tab.

This is a common feature in many text editors but with line numbers and would be very useful to use in combination with the `Vimium` chrome extension. For example, the below image shows relative line numbers in VS Code where the currently selected line is the actual line number and the line numbers for all the other lines are the distance from the currently selected line.

![image](https://github.com/kg8m/chrome-show-tab-numbers/assets/126220639/5ee143f8-cf63-41de-8d2b-cecaeeb6018e)

**Comment by kg8m** [comment]
@RadianttK 
Hi there!

Big thanks for pitching in with the relative numbers idea. I’ve implemented it in Pull Request #53 and #54. I’ve rolled out version v1.3.1 with your suggested feature, and it’s now in the review queue at the Chrome Web Store. Should be live pretty soon!

I’d love to hear what you think about it once it’s out. Feel free to drop your thoughts here, or if something specific comes up, just open a new issue or start a discussion thread.

**Reply by konata-san** [reply]
@kg8m 

Oh, wow thanks! I wish I saw this sooner as this is exactly what I was looking for. I re-installed the extension on my new PC and it's great with the vimium plugin!

The only change I would make is maybe removing the minus and plus signs for the numbers as it takes up additional space and I only need the number to know how far to jump. This is just my personal preference though!

Thanks again!

**Reply by kg8m** [reply]
I have released v1.4.0 and removed the signs `-` and `+` from the relative numbers. The new version will be published after the review.

**Reply by konata-san** [reply]
Thanks so much! I really appreciate it. Looking forward to trying out the new version once it's live. Keep up the great work!
